packetery/src/PacketeryModule/Checkout.php

```diff
@@ -364,8 +364,14 @@ public function getShippingRates(): array {
 		$availableCarriers = $this->carrierRepository->getByCountryIncludingZpoints( $customerCountry );
```

Comment:
Když tuto metodu upravíš tak aby vracela Entity\Carrier[] nebo když nainstancuješ entitu uvnitř toho foreache a použiješ jí, tak najednou máš opakující se kód, který zkoumá to samé jako v administraci. Z toho vyplyne, že to má být v metodě. Třeba $carrierManager->isAvailable($carrier).